Cargo.toml

```diff
@@ -42,6 +42,8 @@ serde_json = "1.0.61"
 [features]
 default = ["macros"]
 
+uselibc = ["libc"]
```

Comment:
There is no need to alias "libc" here; you can simply omit this and use `#[cfg(feature ="libc")]` when needed.

Reply:
Thanks for showing me this!